dg-commons/pom.xml

```diff
@@ -10,20 +10,123 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>dg-commons</artifactId>
+    <groupId>org.finra.datagenerator.commons</groupId>
+    <version>2.2-SNAPSHOT</version>
     <name>Data Generator Common Utils</name>
 
     <dependencies>
+        <!-- Used for base-64 encoding for HTTP basic authentication (which we need to access Jenkins). -->
```

Comment:
Nothing in this repo is. Many of the helpers are not being used by anything in this repo, but are being used internal to FINRA, in GOATS. But none of the helpers are GOATS-specific or FINRA-specific so I decided to commonize them in case they are useful to anyone else.

In this case, GOATS uses this to ensure the latest build is installed prior to testing.

-Samer